shoes-core/spec/shoes/shared_examples/dsl/arrow.rb

```diff
@@ -0,0 +1,96 @@
+shared_examples_for "arrow dimensions" do
+  it "makes a Shoes::Arrow" do
+    expect(arrow).to be_an_instance_of(Shoes::Arrow)
+  end
+
+  it "sets proper dimensions" do
+    expect(arrow.left).to eq(left)
+    expect(arrow.top).to eq(top)
+    expect(arrow.width).to eq(width)
+  end
+end
+
+shared_examples_for "arrow DSL method" do
+  let(:left)   { 40 }
+  let(:top)    { 30 }
+  let(:width)  { 12 }
+
+  context "no arguments" do
+    subject(:arrow) { dsl.arrow }
+
+    let(:left)  { 0 }
+    let(:top)   { 0 }
+    let(:width) { 0 }
+
+    include_examples "arrow dimensions"
```

Comment:
huh, never seen this. I think we always used to do `it_behaves_like` but this seems fine with me as well also I don't think we NEED conformity here :)